src/vs/workbench/parts/stats/node/workspaceStats.ts

```diff
@@ -343,36 +399,90 @@ export class WorkspaceStats implements IWorkbenchContribution {
 				tags['workspace.android.cpp'] = true;
 			}
 
-			const packageJsonPromises = !nameSet.has('package.json') ? [] : folders.map(workspaceUri => {
-				const uri = workspaceUri.with({ path: `${workspaceUri.path !== '/' ? workspaceUri.path : ''}/package.json` });
-				return this.fileService.resolveFile(uri).then(() => {
-					return this.fileService.resolveContent(uri, { acceptTextOnly: true }).then(content => {
-						try {
-							const packageJsonContents = JSON.parse(content.value);
-							if (packageJsonContents['dependencies']) {
-								for (let module of ModulesToLookFor) {
-									if ('react-native' === module) {
-										if (packageJsonContents['dependencies'][module]) {
-											tags['workspace.reactNative'] = true;
-										}
-									} else {
-										if (packageJsonContents['dependencies'][module]) {
-											tags['workspace.npm.' + module] = true;
-										}
-									}
-								}
-							}
-						}
-						catch (e) {
-							// Ignore errors when resolving file or parsing file contents
-						}
+			function getFilePromises(filename, fileService, contentHandler): TPromise<void>[] {
+				return !nameSet.has(filename) ? [] : folders.map(workspaceUri => {
+					const uri = workspaceUri.with({ path: `${workspaceUri.path !== '/' ? workspaceUri.path : ''}/${filename}` });
+					return fileService.resolveFile(uri).then(() => {
+						return fileService.resolveContent(uri, { acceptTextOnly: true }).then(contentHandler);
+					}, err => {
+						// Ignore missing file
 					});
-				}, err => {
-					// Ignore missing file
 				});
+			}
+
+			const requirementsTxtPromises = getFilePromises('requirements.txt', this.fileService, content => {
+				try {
+					let dependencies: string[] = content.value.split('\n');
+					if (!tags['workspace.py.any-azure']) {
+						tags['workspace.py.any-azure'] = WorkspaceStats.searchArray(dependencies, /azure/i);
+					}
+
+					for (let dependency of dependencies) {
+						// Dependencies in requirements.txt can have 3 formats: `foo==3.1, foo>=3.1, foo`
+						const format1 = dependency.split('==')[0];
+						const format2 = dependency.split('>=')[0];
```

Comment:
`format1` will always have something. `foo.split('==')` gives an array with a single string of "foo"